packages/components/psammead-navigation/src/DropdownNavigation/index.test.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { shouldMatchSnapshot } from '@bbc/psammead-test-helpers';
+import { latin } from '@bbc/gel-foundations/scripts';
+import {
+  CanonicalDropdown,
+  DropdownUl,
+  DropdownNavigationLi,
+  CanonicalHamburgerMenu,
+} from './index';
+import pidginNavData from '../../testHelpers/pidgin';
+
+describe('Canonical dropdown navigation', () => {
+  shouldMatchSnapshot(
+    'should render correctly',
+    <CanonicalDropdown announcedText="Menu" onClose={() => {}}>
```

Comment:
Should there also be a test to make sure that the onClose handler runs when the cross icon is clicked?

Reply:
I thought the same thing and looked into this. We can do something similar to https://github.com/bbc/psammead/blob/latest/packages/components/psammead-media-player/src/Placeholder/index.test.jsx#L58.

```jsx
  it('should click click', () => {
    const mockOnClick = jest.fn();
    const { container } = render(
      <CanonicalHamburgerMenu announcedText="Menu" onOpen={mockOnClick} />,
    );
    const button = getByRole(container, 'button');
    fireEvent.click(button);
    expect(mockOnClick).toBeCalledTimes(1);
  });
```

Reply:
Good point, will add that